resteasy-core/src/main/java/org/jboss/resteasy/core/ContextParameterInjector.java

```diff
@@ -38,13 +37,39 @@
 @SuppressWarnings("unchecked")
 public class ContextParameterInjector implements ValueInjector
 {
+   private static Constructor<?> constructor;
+
    private Class<?> rawType;
    private Class<?> proxy;
    private ResteasyProviderFactory factory;
    private Type genericType;
    private Annotation[] annotations;
    private volatile boolean outputStreamWasWritten = false;
 
+   static
+   {
+      try
+      {
+         Class.forName("javax.servlet.http.HttpServletResponse");
+         Class<?> clazz = Class.forName("org.jboss.resteasy.core.ContextServletOutputStream");
+         constructor = clazz.getConstructor(ContextParameterInjector.class, OutputStream.class);
+      }
+      catch (Throwable e)
+      {
+        //
+      }
+   }
```

Comment:
I'm a little confused on why we're doing it this way. I don't think we need to use reflection to create the wrapper.